### ISSUE TITLE
Module FS not found when running with webpack

### ISSUE THREAD

**Issue by Ericson578** [ryanbahniuk/scss-to-json#10]
I'm seeing these errors when trying to use the example usage code for scss-to-json with webpack:

WARNING in ./~/node-sass/lib/index.js
Critical dependencies:
23:14-43 the request of a dependency is an expression
 @ ./~/node-sass/lib/index.js 23:14-43

ERROR in ./~/scss-to-json/src/processor.js
Module not found: Error: Cannot resolve module 'fs' in /Users/ericsonwilkinson/code/avui/ui-usm/node_modules/scss-to-json/src
 @ ./~/scss-to-json/src/processor.js 3:9-22

ERROR in ./~/node-sass/lib/extensions.js
Module not found: Error: Cannot resolve module 'fs' in /Users/ericsonwilkinson/code/avui/ui-usm/node_modules/node-sass/lib
 @ ./~/node-sass/lib/extensions.js 6:9-22

ERROR in ./~/node-sass/lib/render.js
Module not found: Error: Cannot resolve module 'fs' in /Users/ericsonwilkinson/code/avui/ui-usm/node_modules/node-sass/lib
 @ ./~/node-sass/lib/render.js 6:9-22

ERROR in ./~/mkdirp/index.js
Module not found: Error: Cannot resolve module 'fs' in /Users/ericsonwilkinson/code/avui/ui-usm/node_modules/mkdirp
 @ ./~/mkdirp/index.js 2:9-22

**Comment by egervari** [comment]
I have the same issue

**Comment by dubcdr** [comment]
yeah i see as well. i think fsevents only works on macs. so us windows and linux folks are sol

**Comment by mancristiana** [comment]
I tried using this package inside a react app and I got the same issue.
